clang/test/SemaSYCL/sycl_wg_scope.cpp

```diff
@@ -58,6 +58,8 @@ template <typename T> class [[__sycl_detail__::wg_scope]] B12 { // #B12
   B12() = default;
   ~B12() = default;
   T obj;
+  operator T&() {return obj;}
+  T *operator&() noexcept { return &obj; }
```

Comment:
It isn't clear to me what these operators are intended to exercise. The conversion operator doesn't appear to be used anywhere.